person.rb

```diff
@@ -25,4 +26,9 @@ def correct_name
   def of_age?
     @age >= 18
   end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.person = self
+  end
```

Comment:
- [ ] Same here,  This method suppose to create rentals and the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association, So please fix that.